eos/db/saveddata/fit.py

```diff
@@ -106,7 +106,7 @@ def __repr__(self):
                collection_class=HandledProjectedModList,
                cascade='all, delete, delete-orphan',
                single_parent=True,
-               primaryjoin=and_(modules_table.c.fitID == fits_table.c.ID, modules_table.c.projected == True)),
+               primaryjoin=and_(modules_table.c.fitID == fits_table.c.ID, modules_table.c.projected is True)),
```

Comment:
Has this been tested? A previous commit reverted something like this because sql alchemy doesn't support it.